.buildkite/build-and-test-pipeline.js

```diff
@@ -36,44 +36,11 @@ function buildStepForNix (platform) {
   };
 }
 
-function testStepForWindows () {
-  return {
-    label: ':windows: :electron: Test',
-    timeout_in_minutes: 60,
-    command: '.\\src\\buildkite-upload-script\\electron\\.buildkite\\windows\\run-tests',
-    agents: [
-      'os=windows',
-      'queue=electron-build-v7'
-    ]
-  };
-}
-
-function testStepForNix (platform) {
-  return {
-    label: `:${platform}: :electron: Test`,
-    timeout_in_minutes: 60,
-    command: [`.buildkite/nix/run-tests.sh ${platform}`],
-    agents: [
-      `os=${platform}`,
-      'queue=electron-build-v7.2'
-    ]
-  };
-}
-
 function generateBuildPipeline () {
-  // Do not run the pipeline if a PR has not been raised yet
-  // if (process.env.BUILDKITE_PULL_REQUEST === 'false') {
```

Comment:
why don't we need this?

Reply:
Just wanted to manually start the pipeline build, it will not spoil our pipelines. Since releasing electron binaries happens once a while, it can be considered as good to have